### ISSUE TITLE
Scout ablation C2-C5, C7-C10 (single-seed, preliminary, 70 cases)

### ISSUE THREAD

**Issue by wrockey** [wrockey/vmat-diffusion#65]
## Purpose

Single-seed (seed42) scouts of the 8 remaining ablation conditions on the preliminary 70-case dataset. These are NOT publishable results — they validate the full C1-C10 pipeline and give early directional signal before committing to 30 runs on Argon with the final ~91-case dataset.

## Conditions

**Addition arm (MSE + single component):**
| ID | Condition | Loss flags |
|----|-----------|------------|
| C2 | MSE + Gradient | `--use_gradient_loss` |
| C3 | MSE + DVH | `--use_dvh_loss` |
| C4 | MSE + Structure | `--use_structure_weighted` |
| C5 | MSE + AsymPTV | `--use_asymmetric_ptv` |

**Leave-one-out arm (Full combined minus one):**
| ID | Condition | Removed component |
|----|-----------|-------------------|
| C7 | Full − Gradient | gradient loss |
| C8 | Full − DVH | DVH loss |
| C9 | Full − Structure | structure-weighted loss |
| C10 | Full − AsymPTV | asymmetric PTV loss |

## Protocol

- Seed: 42 only (preliminary)
- Dataset: 70 cases (v2.3, preliminary)
- Epochs: 200 (early stopping patience=50)
- All other hyperparameters identical to baseline_v23 / combined_loss_2.5:1
- Sequential execution on RTX 3090 (~40 hours total)
- Automated script: `scripts/run_ablation_scouts.sh`

## Tracking

Progress tracked via comments on this issue. Each run posts start/finish with key metrics.

## Depends on

- C1 baseline_v23 (3-seed complete) ✅
- C6 combined_loss_2.5:1 (3-seed complete) ✅
- #43 (ablation design)

---

## Documentation Checklist (REQUIRED before closing this issue)

The runner script (`scripts/run_ablation_scouts.sh`) automates training, inference, and partial documentation. The remaining items must be completed manually.

### Automated by script
- [ ] Training + inference for all 8 conditions (C2-C5, C7-C10)
- [ ] Per-condition results posted as comments on this issue
- [ ] Environment snapshots saved (`runs/*/environment_snapshot.txt`)
- [ ] `notebooks/EXPERIMENTS_INDEX.md` updated with 8 new rows
- [ ] Comparison figures generated (`runs/ablation_scouts/figures/`)
- [ ] Full C1-C10 summary table posted to this issue

### Manual post-processing
- [ ] **Review auto-generated EXPERIMENTS_INDEX.md entries** — verify metrics are correct
- [ ] **Create combined notebook**: `notebooks/YYYY-MM-DD_ablation_scouts_preliminary.ipynb`
  - Use `notebooks/TEMPLATE_experiment.ipynb` as base
  - **All 10 sections required** (per CLAUDE.md), adapted for multi-condition analysis:
    1. Overview — objective, hypothesis, key findings across all 8 conditions
    2. What Changed — table showing each condition's loss configuration vs C1/C6
    3. Reproducibility — git hash, seed, CLI commands for each condition
    4. Dataset — same 70 cases (v2.3), test split, preprocessing version
    5. Model & Training — same BaselineUNet3D; only loss components differ
    6. Results — load figures from `runs/ablation_scouts/figures/`; add captions
    7. Statistical Analysis — per-case paired comparisons (Wilcoxon, Holm-Bonferroni). Note: single-seed so treat as preliminary
    8. Cross-Experiment Comparison — table with all 10 conditions + prior experiments
    9. Conclusions — which components matter, which are redundant, implications for final Argon runs
    10. Artifacts — paths to all run dirs, prediction dirs, figure dir
  - **No placeholders** — all sections filled, all figures captioned
  - **Status: Preliminary** (single-seed, 70-case dataset)
- [ ] **Review comparison figures** — add captions to notebook, check axes/labels
- [ ] **Commit all documentation**:
  ```bash
  git add scripts/generate_ablation_scouts_figures.py
  git add notebooks/YYYY-MM-DD_ablation_scouts_preliminary.ipynb
  git add notebooks/EXPERIMENTS_INDEX.md
  git commit -m "docs: Add ablation scout preliminary results (C2-C5, C7-C10)"
  ```
- [ ] **Update pinned issue #63** if results change the project narrative (e.g., a single component matches full combined → simpler model viable)
- [ ] **Update #43** with scout findings — note any conditions that need attention in final Argon runs

### Quality gates (do not close this issue until ALL are met)
- [ ] All 8 conditions trained and evaluated successfully
- [ ] EXPERIMENTS_INDEX.md has 8 new rows with correct metrics
- [ ] Combined notebook exists with all 10 sections filled (no `[UPDATE]`, no `None`, no placeholders)
- [ ] Figures generated and referenced in notebook with captions
- [ ] All artifacts committed and pushed
- [ ] This issue has the full C1-C10 comparison table in comments

**Comment by wrockey** [comment]
### C2 (MSE+Gradient) — Inference failed

**Comment by wrockey** [comment]
### C3 (MSE+DVH) — Training started
```
/home/wrockey/miniforge3/envs/vmat-diffusion/bin/python scripts/train_baseline_unet.py --data_dir /home/wrockey/data/processed_npz --epochs 200 --seed 42 --num_workers 2 --exp_name C3_dvh_only --use_dvh_loss --dvh_loss_weight 0.5
```
Started: 2026-03-07 23:27

**Comment by wrockey** [comment]
### C5 (MSE+AsymPTV) — Training failed

**Comment by wrockey** [comment]
### C7 (Full-Gradient) — Training failed

**Comment by wrockey** [comment]
### C8 (Full-DVH) — Training started
```
/home/wrockey/miniforge3/envs/vmat-diffusion/bin/python scripts/train_baseline_unet.py --data_dir /home/wrockey/data/processed_npz --epochs 200 --seed 42 --num_workers 2 --exp_name C8_full_no_dvh --use_gradient_loss --gradient_loss_weight 0.1 --use_structure_weighted --structure_weighted_weight 1.0 --use_asymmetric_ptv --asymmetric_ptv_weight 1.0 --asymmetric_underdose_weight 2.5 --asymmetric_overdose_weight 1.0 --use_uncertainty_weighting --calibration_json /home/wrockey/data/processed_npz/loss_normalization_calib.json
```
Started: 2026-03-07 23:28

**Comment by wrockey** [comment]
### C9 (Full-Structure) — Training failed
